source/Octopus.Tentacle/Services/Scripts/ScriptServiceV3Alpha.cs

```diff
@@ -116,6 +116,9 @@ public async Task CompleteScriptAsync(CompleteScriptCommandV3Alpha command, Canc
 
             var workspace = workspaceFactory.GetWorkspace(command.ScriptTicket);
             await workspace.Delete(cancellationToken);
+
+            if (runningScript?.Process is not null)
+                await runningScript.Process.Cleanup(cancellationToken);
```

Comment:
As there is no longer a TTL for automatic cleanup, we now ask the process (`KubernetesRunningPod`) to cleanup when `CompleteScript` is called